src/js/claims-status/components/appeals-v2/Alerts.jsx

```diff
@@ -1,21 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Alert from './Alert';
+import { getAlertContent } from '../../utils/appeals-v2-helpers';
 
-const Alerts = (props) => {
-  if (typeof props.alerts === 'undefined' || props.alerts.length === 0) {
+const Alerts = ({ alerts }) => {
```

Comment:
While I'm thinking about it, what do you think about renaming this to `AlertList` to avoid confusion between `Alerts` and `Alert`?

Doesn't have to be done in this PR. I'm just wondering if that'd be helpful.